scene/audio/audio_stream_player.cpp

```diff
@@ -149,7 +149,11 @@ void AudioStreamPlayer::_notification(int p_what) {
 
 	if (p_what == NOTIFICATION_INTERNAL_PROCESS) {
 
-		if (!active || (setseek < 0 && !stream_playback->is_playing())) {
+		if (!active || setstop) {
+			active = false;
+			set_process_internal(false);
```

Comment:
@Sslaxx yeah a new `stopped` signal could be added:
```suggestion
			set_process_internal(false);
			emit_signal("stopped")
```
But I think this can be added in a separate PR without much hassle.